gnosis/safe/addresses.py

```diff
@@ -595,6 +595,12 @@
         ("0x3E5c63644E683549055b9Be8653de26E0B4CD36E", 2362236, "1.3.0+L2"),
         ("0xd9Db270c1B5E3Bd161E8c8503c55cEABeE709552", 2362238, "1.3.0"),
     ],
+    EthereumNetwork.PGN_MAINNET: [
+        ("0x3E5c63644E683549055b9Be8653de26E0B4CD36E", 344345, "1.3.0+L2"),
+    ],
```

Comment:
1.3.0 contract was not deployed there? 0x3E5c63644E683549055b9Be8653de26E0B4CD36E

Reply:
I've followed the README instructions, and from what I understood I should only fill 1.3.0 L2 version contract address and not 1.3.0 contract address.
So it seems I misunderstood, I will add the 1.3.0 contract address too.

Reply:
@moisses89 added 1.3.0 contract